python/vyos/remote.py

```diff
@@ -180,7 +181,14 @@ def transfer_sftp(mode, local_path, hostname, remote_path,\
                   source=None, progressbar=False):
     sock = None
     if source:
-        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        # Check if the given string is an IPv6 address.
+        try:
+            ipaddress.IPv6Address(source)
```

Comment:
Why not use vyos.util.is_ipv6()?

Reply:
Oh, it's in `vyos.validate`, I only checked `vyos.util`. I'll swap it.